src/org/opensolaris/opengrok/configuration/RuntimeEnvironment.java

```diff
@@ -1024,6 +1045,14 @@ public void setScopesEnabled(boolean scopesEnabled) {
         threadConfig.get().setScopesEnabled(scopesEnabled);
     }
 
+    public boolean isProjectsEnabled() {
+        return threadConfig.get().isProjectsEnabled();
+    }
```

Comment:
- shouldn't we use this in Util.dumpConfiguration?
- shouldn't we use this in env.hasProjects?